pkg/loop/internal/pb/plugin_service.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb";
+
+package loop;
+
+service PluginService {
+  rpc NewPluginServiceFactory (NewPluginServiceFactoryRequest) returns (NewPluginServiceFactoryReply) {}
+}
+
+message PluginServiceConfig {
+  string providerType = 1;
+  string command = 2;
+  string pluginName = 3;
+  bytes pluginConfig = 4;
+}
+
+// NewPluginServiceFactoryRequest has arguments for [github.com/smartcontractkit/chainlink-relay/pkg/loop.PluginService.NewPluginServiceFactory].
+message NewPluginServiceFactoryRequest {
+  uint32 providerID = 1;
+  uint32 errorLogID = 2;
+  PluginServiceConfig pluginServiceConfig = 3;
+}
+
+// NewPluginServiceFactoryReply has return arguments for [github.com/smartcontractkit/chainlink-relay/pkg/loop.PluginService.NewPluginServiceFactory].
+message NewPluginServiceFactoryReply {
+  uint32 reportingPluginFactoryID = 1;
```

Comment:
does the field name need all the extra qualifies? seems like `id` ought to be enough: `NewPluginServiceFactoryReply.ID`

wdyt?

Reply:
Yes, good point :)